### ISSUE TITLE
Problem creating the super user and logging in the framework (with clean install)

### ISSUE THREAD

**Issue by mrrobotnl** [intelowlproject/IntelOwl#1650]
## What happened
I got a problem, I got the framework up and running the only problem I have has is that I cannot create a superuser. 
## Environment
1. OS: Ubuntu 22.04
2. IntelOwl version: 4.2.3

##  What did you expect to happen
Hey, 
I got a problem, I got the framework up and running the only problem I have has is that I cannot create a superuser. 
I tried the following command: 
docker exec -ti intelowl_uwsgi python3 manage.py createsuperuser

But I got an error, is there a new way to set a user? 

I use the most current version: V4.2.3 and did walked the instruction that is located here: https://intelowl.readthedocs.io/en/latest/Installation.html#requirements 

## Error messages and logs
  File "/opt/deploy/intel_owl/manage.py", line 25, in <module>
    main()
  File "/opt/deploy/intel_owl/manage.py", line 21, in main
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 402, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/management/commands/createsuperuser.py", line 88, in execute
    return super().execute(*args, **options)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 443, in execute
    self.check()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 475, in check
    all_issues = checks.run_checks(
  File "/usr/local/lib/python3.9/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/usr/local/lib/python3.9/site-packages/drf_recaptcha/checks.py", line 18, in recaptcha_system_check
    raise ImproperlyConfigured("settings.DRF_RECAPTCHA_SECRET_KEY must be set.")
django.core.exceptions.ImproperlyConfigured: settings.DRF_RECAPTCHA_SECRET_KEY must be set.

**Comment by mlodic** [comment]
hey!

as a workaround, please populate in your `env_file_app` the variable `RECAPTCHA_SECRET_KEY_IO_LOCAL` with a fake value, restart the application and let me know how it goes

**Comment by mrrobotnl** [comment]
Hey Mlodic, it did totally work thanks

**Comment by dipakparmar** [comment]
For me, it required a real key to work.

**Comment by kittrCZ** [comment]
Had the same problem. I think the documentation has to be updated.

**Comment by mrrobotnl** [comment]
> Had the same problem. I think the documentation has to be updated.

Yes, the fix that Mlodic mentioned worked for me, you will need to go to the env_file_app configuration document and add “Fake_value” to the RECAPTCHA_SECRET_KEY_IO_LOCAL.
